data/193/046/663/3/1930466633.geojson

```diff
@@ -0,0 +1,82 @@
+{
+  "id": 1930466633,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"2022-01-01",
```

Comment:
Not a blocker, but in some of these new records, I see these properties:

- "date:inception_lower"
- "date:inception_upper"

though they're not present in others.

Reply:
the basic inception is from the Python exportify, but if I then also later run the Go exportify tool on the same "new" record it'll add the lower and upper. So annoying but "eh"?

Reply:
Agree, not a blocker.. though this should also work with Python? https://github.com/whosonfirst/py-mapzen-whosonfirst-export/blob/master/mapzen/whosonfirst/export/__init__.py#L214-L215